app/src/main/java/com/ichi2/apisample/helper/AnkiDroidHelper.java

```diff
@@ -323,21 +331,91 @@ public Long addNewDeck(String deckName) {
 
     // @todo: refactor once new version release of "com.ichi2.anki.api" is available
     public String addFileToAnkiMedia(String uriString) {
+        Uri uri = Uri.parse(uriString);
+        mContext.grantUriPermission("com.ichi2.anki", uri, Intent.FLAG_GRANT_READ_URI_PERMISSION);
```

Comment:
I think there should be a special constant for "com.ichi2.anki" (package name or something like that)
if not, please create/move it somewhere